.azure-pipelines-gh-pages.yml

```diff
@@ -7,7 +7,7 @@ trigger:
 
 jobs:
   - job: build_and_publish_docs
-    container: ccfciteam/ccf-ci:oe0.17.1-focal-clang10
+    container: ccfciteam/ccf-ci:oe0.17.1-focal-driverless
```

Comment:
We never installed the driver in containers, so this is a slightly odd name, but not a problem.

Reply:
Yes, you're right. I'll update the name of the image.